lib/web/css/source/lib/variables/_buttons.less

```diff
@@ -103,3 +104,4 @@
 @button__font-size__s: 11px;
 @button__line-height__s: @button__font-size__s + 1;
 @button__padding__s: @indent__xs 8px;
+
```

Comment:
we should use unitless value for line-height when possible. In this we set default line height for button init value one

Reply:
@Leland what do you think about this